### PR TITLE
[es] localize 'Containerization'

### LINKED ISSUE #577: [Spanish] localize 'Containerization'
Localize the `content/en/containerization.md` document to Spanish

### PR DESCRIPTION
**Pull request by RedwineOS**
### Describe your changes
Localization in spanish for containerization.

### Related issue number or link (ex: `resolves #issue-number`)
Resolves #577 

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco).

### CODE REVIEW

**Review by bconfiden2 — COMMENTED** [review]
According to [build error](https://github.com/cncf/glossary/runs/8209221669?check_suite_focus=true), some words such as `Container` and `Initiative` are English, and the other words such as `hipervisor` and `reprogrmados` are Spanish.
So I wonder why these errors occur.
Should these words be in file `.wordlist.txt` ??
I'm curious about how `spellcheck.yml`-Spanish l10n team's CI-works!! :pleading_face: :pleading_face:

**Comment by RedwineOS** [comment]
> According to [build error](https://github.com/cncf/glossary/runs/8209221669?check_suite_focus=true), some words such as `Container` and `Initiative` are English, and the other words such as `hipervisor` and `reprogrmados` are Spanish. So I wonder why these errors occur. Should these words be in file `.wordlist.txt` ?? I'm curious about how `spellcheck.yml`-Spanish l10n team's CI-works!! 🥺 🥺

Those words have to be present in the `.wordlist` file so that the job does not fail.

**Comment by ramrodo** [comment]
Colocar la opción `-f` para que sea incase sensitive:

```bash
sort -f -o content/es/.wordlist.txt content/es/.wordlist.txt
```

File: content/es/.wordlist.txt [hunk 1]
```diff
@@ -26,6 +26,7 @@ Business
 BVS
 BY
 calendar
+Canary
 canary
```
Comment (electrocucaracha) [inline content/es/.wordlist.txt:30]: Dado que la lista de palabras prioriza las mayúsculas y para evitar tener duplicados, sugiero
```suggestion
Canary
```

File: content/es/.wordlist.txt [hunk 2]
```diff
@@ -53,6 +54,7 @@ creating
 CRI
 Daniel
 declarativamente
+Deployment
 deployment
```
Comment (electrocucaracha) [inline content/es/.wordlist.txt:58]: ditto: línea 29
```suggestion
Deployment
```

File: content/es/.wordlist.txt [hunk 3]
```diff
@@ -70,13 +72,16 @@ extensibilidad
 Feedback
 first
 Foster
+Foster
```
Comment (electrocucaracha) [inline content/es/.wordlist.txt:75]:
```suggestion
```

File: content/es/.wordlist.txt [hunk 4]
```diff
@@ -70,13 +72,16 @@ extensibilidad
...
 foundation
 framework
 frontend
 Garcia
 gateway
 GCP
+Git
 git
```
Comment (electrocucaracha) [inline content/es/.wordlist.txt:83]: ditto: línea 29
```suggestion
Git
```

File: content/es/.wordlist.txt [hunk 5]
```diff
@@ -70,13 +72,16 @@ extensibilidad
...
+GitHub
 github
```
Comment (electrocucaracha) [inline content/es/.wordlist.txt:85]: ditto: línea 29
```suggestion
GitHub
```

File: content/es/.wordlist.txt [hunk 6]
```diff
@@ -187,5 +195,6 @@ Word
 work
 www
 YAML
+YAML
```
Comment (electrocucaracha) [inline content/es/.wordlist.txt:198]:
```suggestion
```

File: content/es/containerization.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+tags: ["", "", ""]
+---
+
+## ¿Qué es?
+
+La contenerización es el proceso que consiste en empaquetar una aplicación y sus dependencias en una [imagen de contenedor](./container-image.md). La construcción del contenedor requiere del seguimiento del estándar [Open Container Initiative](https://opencontainers.org) (OCI). Mientras la salida de este proceso sea un contenedor de imagen que se adhiera a dicho estándar, la herramienta de contenerización usada no es relevante.
+
+## Problema que aborda
+
+Antes de que los contenedores se convirtieran en prevalentes, las organizaciones dependían de máquinas virtuales para orquestar múltiples aplicaciones en una única computadora. Las máquinas virtuales son significativamente más pesadas que los contenedores y requieren de un hipervisor para su funcionamiento. Debido al almacenamiento, copias de seguridad y transferencia de estas plantillas de máquinas virtuales, la creación de las mismas también se trata de un proceso lento. Adicionalmente, las máquinas virtuales pueden sufrir variaciones en la configuración, lo cuál contradice el principio de inmutabilidad.
```
Comment (electrocucaracha) [inline content/es/containerization.md:14]:
```suggestion
Antes de que los contenedores se convirtieran relevantes, las organizaciones dependían de máquinas virtuales para orquestar múltiples aplicaciones en una única computadora. Las máquinas virtuales son significativamente más pesadas que los contenedores y requieren de un hipervisor para su funcionamiento. Debido al almacenamiento, copias de seguridad y transferencia de estas plantillas de máquinas virtuales, la creación de las mismas también se trata de un proceso lento. Adicionalmente, las máquinas virtuales pueden sufrir variaciones en la configuración, lo cuál contradice el principio de inmutabilidad.
```
Si esta sugerencia es aceptada, será necesario eliminar `prevalentes` del wordlist

File: content/es/containerization.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
+---
+title: Contenerización
+status: Completed
+category: Tecnología
+tags: ["", "", ""]
+---
+
+## ¿Qué es?
+
+La contenerización es el proceso que consiste en empaquetar una aplicación y sus dependencias en una [imagen de contenedor](./container-image.md). La construcción del contenedor requiere del seguimiento del estándar [Open Container Initiative](https://opencontainers.org) (OCI). Mientras la salida de este proceso sea un contenedor de imagen que se adhiera a dicho estándar, la herramienta de contenerización usada no es relevante.
```
Comment (electrocucaracha) [inline content/es/containerization.md:10]:
```suggestion
La contenerización es el proceso que consiste en empaquetar una aplicación y sus dependencias en una [imagen de contenedor](/es/container-image.md). La construcción del contenedor requiere del seguimiento del estándar [Open Container Initiative](https://opencontainers.org) (OCI). Mientras la salida de este proceso sea un contenedor de imagen que se adhiera a dicho estándar, la herramienta de contenerización usada no es relevante.
```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]

**Comment by electrocucaracha** [comment]
> According to [build error](https://github.com/cncf/glossary/runs/8209221669?check_suite_focus=true), some words such as `Container` and `Initiative` are English, and the other words such as `hipervisor` and `reprogrmados` are Spanish. So I wonder why these errors occur. Should these words be in file `.wordlist.txt` ?? I'm curious about how `spellcheck.yml`-Spanish l10n team's CI-works!! 🥺 🥺

@bconfiden2 those English word are exceptions that we have to add into our [word list](https://github.com/cncf/glossary/blob/dev-es/content/es/.wordlist.txt), [this](https://github.com/cncf/glossary/blob/dev-es/content/es/.spellcheck.yml#L21) is our configuration file which uses Aspell Spanish dictionary on [any](https://github.com/cncf/glossary/blob/dev-es/.github/workflows/es-spellcheck.yml) PR to `dev-es`

File: content/es/containerization.md (hunk 7 above)
Comment (ramrodo) [reply content/es/containerization.md:14]: Creo que debería de ser: _Antes de que los contenedores se convirtieran **en relevantes**_:

```suggestion
Antes de que los contenedores se convirtieran en relevantes, las organizaciones dependían de máquinas virtuales para orquestar múltiples aplicaciones en una única computadora. Las máquinas virtuales son significativamente más pesadas que los contenedores y requieren de un hipervisor para su funcionamiento. Debido al almacenamiento, copias de seguridad y transferencia de estas plantillas de máquinas virtuales, la creación de las mismas también se trata de un proceso lento. Adicionalmente, las máquinas virtuales pueden sufrir variaciones en la configuración, lo cuál contradice el principio de inmutabilidad.
```

File: content/es/containerization.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+
+## ¿Cómo ayuda?
+
+Al contrario que con las máquinas virtuales tradicionales, las imágenes de contenedor son ligeras y el proceso de contenerización requiere de una lista exhaustiva de dependencias. Este archivo puede ser versionado y el proceso de construcción automatizado, permitiendo a una organización centrarse en otras prioridades mientras los procesos autónomos son los encargados de la construcción. Una imagen de contenedor se almacena junto a un identificador único que está ligado a un contenido y configuración exactos. Conforme los contenedores son programados y reprogramados siempre van a ser revertidos a su estado inicial, lo cuál elimina variaciones en la configuración.
```
Comment (krol3) [inline content/es/containerization.md:18]:
```suggestion
Al contrario que con las máquinas virtuales tradicionales, las imágenes de contenedor son ligeras y el proceso de contenerización requiere de una lista exhaustiva de dependencias. Este archivo puede ser versionado y el proceso de construcción automatizado, permitiendo a una organización centrarse en otras prioridades mientras que los procesos automatizados se encargan de la construcción. Una imagen de contenedor se almacena junto a un identificador único que está vinculado a un contenido y configuración exacta. Conforme los contenedores son programados y reprogramados siempre van a ser revertidos a su estado inicial, lo cuál elimina variaciones en la configuración.
```

**Review by krol3 — COMMENTED** [review]
LGTM small changes

**Review by electrocucaracha — APPROVED** [review]

**Pull request merged**